### ISSUE TITLE
Boost warning made silent (temporarily)

### ISSUE THREAD

**Issue by janetournois** [CGAL/cgal#4831]
## Issue Details

In PR #4803 [testsuite](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.1-Ic-190/Tetrahedral_remeshing/TestReport_Blake_Windows_MSVCPreview-Release-64bits.gz) we have met this warning from Boost:
```
"C:\3rdPartyLibs\boost_1_73_0\boost/bimap/detail/bimap_core.hpp(408): warning C4996:
'std::allocator<void>::rebind<
   boost::bimaps::relation::mutant_relation<
     boost::bimaps::tags::tagged<
       const CGAL::internal::CC_iterator<
        CGAL::Compact_container<
          CGAL::Mesh_vertex_base_3<
            GT,MD,CGAL::Regular_triangulation_vertex_base_3<
		      GT,CGAL::Triangulation_ds_vertex_base_3<TDS2>>>,
        CGAL::Default,CGAL::Default,CGAL::Default>,false>,
	  boost::bimaps::relation::member_at::left>,
    boost::bimaps::tags::tagged<const CGAL::internal::CC_iterator<
	  CGAL::Compact_container<
		CGAL::Mesh_vertex_base_3<
		  GT,MD,CGAL::Regular_triangulation_vertex_base_3<
	        GT,CGAL::Triangulation_ds_vertex_base_3<TDS2>>>,
    CGAL::Default,CGAL::Default,CGAL::Default>,false>,
  boost::bimaps::relation::member_at::right>,int,true>>':
warning STL4010: Various members of std::allocator are deprecated in C++17. 
Use std::allocator_traits instead of accessing these members directly. 
You can define _SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING 
or _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS to acknowledge that you have received this warning."
```

We have made him silent by defining  `_SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING`  in `CGAL/config.h`.

It seems that it will be fixed in the next release of Boost, so we should do something to improve this solution.

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Windows
* Compiler: msvc 2019 Preview
* Release or debug mode:
* Specific flags used (if any):
* CGAL version: master as 5.1-beta
* Boost version: 1.73.0
* Other libraries versions if used (Eigen, TBB, etc.):

**Comment by janetournois** [comment]
Note this issue should not be closed when PR #4803 is merged.

**Comment by sloriot** [comment]
Warning is gone
